Makefile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION?=8.2.2
+VERSION?=8.2.0
```

Comment:
Upgrading Redis here isn't sufficient, we have multiple places (such as [`pipeline.properties`](https://github.com/spring-projects/spring-data-build/blob/main/ci/pipeline.properties#L17)) that require an upgrade.

It's likely easier to upgrade the Redis target version first before introducing new commands. In any case, the Redis 8.2 upgrade was done via #3248